File: chepa-desktop/ChepaMotos/ChepaMotos/Models/Vehicle.cs
using System.Text.Json.Serialization;

namespace ChepaMotos.Models;

public class Vehicle
{
    [JsonPropertyName("id")]
    public long Id { get; set; }

    [JsonPropertyName("plate")]
    public string Plate { get; set; } = string.Empty;

    [JsonPropertyName("model")]
    public string Model { get; set; } = string.Empty;
}

File: chepa-api/src/main/java/com/chepamotos/model/Vehicle.java
package com.chepamotos.model;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Entity
@Table(name = "vehicle")
public class Vehicle {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "vehicle_id")
    private Long id;

    @Column(name = "plate", nullable = false, unique = true, length = 20)
    private String plate;

    @Column(name = "model", nullable = false, length = 100)
    private String model;

}